chef/cookbooks/crowbar/templates/default/crowbar-post-upgrade.sh.erb

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# This does what is necessary directly after upgraded node is rebooted
+
+LOGFILE=/var/log/crowbar/node-upgrade.log
+mkdir -p "`dirname "$LOGFILE"`"
+exec >>"$LOGFILE" 2>&1
+
+set -x
+
+<% if @use_ha %>
+
+echo "Unsetting the pre-upgrade node attribute..."
+
+crm node attribute $(hostname) set pre-upgrade false
+
+<% if @have_drbd %>
+
+if [[ -f /sbin/drbdadm ]] ; then
+    echo "Recreating DRBD metadata after the upgrade to new DRBD version..."
+    drbdadm -- --force create-md all
```

Comment:
this will be called whether it's drbd master or slave. Is this the correct behavior?